src/file-uploader/file-uploader.component.ts

```diff
@@ -125,6 +149,13 @@ export class FileUploader {
 	protected onTouchedCallback: () => void = noop;
 	protected onChangeCallback: (_: Set<FileItem>) => void = noop;
 
+	/**
+	 * Controls the state of the drag and drop file container
+	 */
+	protected dragOver = false;
+
+	protected _fileList = [];
```

Comment:
I'm not totally sure why we have `fileList` _and_ `files`? Is there something we can't do with the `files` Set that merits having both?